pkg/crc/validation/validation.go

```diff
@@ -61,6 +61,7 @@ func ValidateBundlePath(bundlePath string, preset crcpreset.Preset) error {
 	}
 
 	userProvidedBundle := filepath.Base(bundlePath)
+	userProvidedBundle = filepath.Base(userProvidedBundle)
```

Comment:
Why is this no longer `	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)`?

Reply:
@cfergeau https://github.com/code-ready/crc/pull/3315#discussion_r962922556 I think I just don't need this line in this commit itself.